### PR TITLE
Make image prefix configurable in setup_image_vars

### PR DESCRIPTION
**Pull request by kargakis**
[test]

### CODE REVIEW

**Comment by kargakis** [comment]
[test]

**Comment by soltysh** [comment]
LGTM

**Comment by soltysh** [comment]
[merge]

**Comment by kargakis** [comment]
@stevekuznetsov fyi

File: hack/lib/util/environment.sh [hunk 1]
```diff
@@ -285,16 +285,17 @@ readonly -f os::util::environment::setup_server_vars
 #  - export MAX_IMAGES_BULK_IMPORTED_PER_REPOSITORY
 function os::util::environment::setup_images_vars() {
     # Use either the latest release built images, or latest.
+    OS_IMAGE_PREFIX="${OS_IMAGE_PREFIX:-"openshift/origin"}"
```
Comment (stevekuznetsov) [inline hack/lib/util/environment.sh]: Let's isolate this, not the best to copy it here from `hack/common.sh`

Comment (kargakis) [inline hack/lib/util/environment.sh]: done

**Comment by stevekuznetsov** [comment]
[merge]

**Pull request merged**